### ISSUE TITLE
Verify starfish-referenced image datasets on s3

### ISSUE THREAD

**Issue by ambrosejcarr** [spacetx/starfish#524]
Currently we're validating only small experiments, and we don't (automatically) verify that our datasets will work with starfish. 

It would be a good idea to: 
1. Ensure that if starfish validates a dataset, that means it works with starfish
2. Incorporate validate into the starfish CLI, e.g.: `starfish validate`
3. As part of our CI, run validate on the datasets that are "currently referenced" by starfish notebooks or are given as public data links to collaborators. I can provide a list of these datasets.

**Comment by joshmoore** [comment]
Once gh-515 and gh-521 land, I'll add validation for each of the experiment.json files in cloudfront.

**Comment by joshmoore** [comment]
On going pieces of the puzzle:

 * https://github.com/spacetx/starfish/pull/523
 * https://github.com/spacetx/starfish/pull/538
 * https://github.com/spacetx/slicedimage/pull/61

**Comment by ambrosejcarr** [comment]
@joshmoore Is this issue closable? If so, please feel free to close it.

**Comment by joshmoore** [comment]
Closing. I'll add a few low priority follow-ons.
